airbyte-ci/connectors/pipelines/README.md

```diff
@@ -22,17 +22,33 @@ This project requires Python 3.10 and pipx.
 
 The recommended way to install `airbyte-ci` is using pipx. This ensures the tool and its dependencies are isolated from your other Python projects.
 
+If you havent installed pyenv, you can do it with brew:
+
+```bash
+brew update
+brew install pyenv
+```
+
 If you haven't installed pipx, you can do it with pip:
 
 ```bash
+cd airbyte-ci/connectors/pipelines/
+pyenv install # ensure you have the correct python version
 python -m pip install --user pipx
 python -m pipx ensurepath
 ```
 
-Once pipx is installed, navigate to the root directory of the project, then run:
+Once pyenv and pipx is installed then run the following:
 
 ```bash
-pipx install airbyte-ci/connectors/pipelines/ --force
+# ensure you are in the correct directory
+cd airbyte-ci/connectors/pipelines/
+
+# ensure you have the correct python version
+pyenv install
+
+# install airbyte-ci
+pipx install --editable --force .
```

Comment:
We could take this one step further with a shell script we can call like so
```
./install-internal-tool airbyte-ci/connectors/pipelines/
```

e.g.
```sh
#!/bin/bash

# Change to the given directory
cd "$1" || { echo "Directory does not exist"; exit 1; }

# Update brew and install pyenv
brew update
brew install pyenv

# Ensure correct Python version
pyenv install

# Install pipx
python -m pip install --user pipx
python -m pipx ensurepath

# Install ci_credentials
pipx install --editable --force .
```